source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4914,8 +4918,12 @@ export class FilteredChoiceSet {
                 `ac-choiceSetInput-${this._choiceSetId}-choice-${id}`
             );
             if (nextHighlightedChoice) {
-                currentHighlightedChoice?.classList.remove("focused");
-                nextHighlightedChoice.classList.add("focused");
+                currentHighlightedChoice?.classList.remove(
```

Comment:
We will need to remove this constant literal and replace it with a variable. Then, we should link the variable to a relevant property to avoid any unexpected issues. By doing so, we can save a significant amount of effort in re-engineering.

Reply:
Will resolve this conversation and merge PR post further testing with the team to check if any other fixes need to be made.